include/libopencm3/stm32/l4/flash.h

```diff
@@ -1,21 +1,21 @@
 /** @defgroup flash_defines FLASH Defines
- *
- * @ingroup STM32L4xx_defines
  *
  * @brief <b>Defined Constants and Types for the STM32L4xx Flash Control</b>
  *
+ * @ingroup STM32L4xx_defines
+ *
  * @version 1.0.0
  *
- * @author @htmlonly &copy; @endhtmlonly 2016 Benjamin Levine <benjamin@jesco.karoo.co.uk>
+ * @author @htmlonly &copy; @endhtmlonly 2019 Guillaume Meunier <guillaume.meunier@centraliens.net>
  *
- * @date 12 February 2016
+ * @date 2 July 2019
  *
  * LGPL License Terms @ref lgpl_license
- *  */
+ */
 /*
  * This file is part of the libopencm3 project.
  *
- * Copyright (C) 2016 Benjamin Levine <benjamin@jesco.karoo.co.uk>
+ * Copyright (C) 2019 Guillaume Meunier <guillaume.meunier@centraliens.net>
```

Comment:
So as I understand it, you moved this entire file to the common_l4_flash file, where it kept benajamin's copyright (good) and you've then put your own copyright on the empty file remaining?  I would that is, at least poor form, and somewhat disingenuous, but it's probably still _legal_ to copyright the #include line.  I'd lean towards dropping the claim entirely, (Benjamin's moves to the common file) or leaving benjamin's.  Certainly until you've added more meaningful content to this file.

Reply:
Yes, I moved the entire file. I can remove the copyright line as it's only a single #include (in the other files as well).